sm/plugins/tedge_dummy_plugin/tests/tests.rs

```diff
@@ -0,0 +1 @@
+
```

Comment:
This is a trick to get the `tedge_dummy_plugin` build with all tests such it can be used in tests.